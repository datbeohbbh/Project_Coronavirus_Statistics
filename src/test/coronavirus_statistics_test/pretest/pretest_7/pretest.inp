5 12
q 1 2 22
q 2 2 5
? 4 1
? 7
q 1 3 17
q 1 1 22
q 2 3 3
? 4 3
q 1 4 40
? 4 1
? 6
? 4 4