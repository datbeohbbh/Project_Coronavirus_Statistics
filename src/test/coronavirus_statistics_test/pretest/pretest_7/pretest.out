Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 17
-> Has recovered : 5
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 3 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 22
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 14
-> Has recovered : 3
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 2
Quantity: 5 people

Query ? 4 4 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 40
-> Has recovered : 0
-> Was dead : 0

